metabase/metabase
# Migrate timeline forms to formik

Epic https://github.com/metabase/metabase/issues/26178

Changes:
- Adds `TimelineForm` used in timeline modals
- Updates `NewTimelineModal` and `EditTimelineModal` to use this form

How to test:
- Go to a collection and click on the calendar icon in the top right corner
- Click on `...` -> New timeline
- Make sure it is possible to create a new timeline via the form
- Click on `...` -> Edit timeline details
- Make sure it is possible to update the timeline via the form

<img width="839" alt="Screenshot 2022-11-04 at 15 46 37" src="https://user-images.githubusercontent.com/8542534/199987714-b5787336-72b0-4c21-b86c-614270297e1c.png">
<img width="661" alt="Screenshot 2022-11-04 at 15 46 03" src="https://user-images.githubusercontent.com/8542534/199987753-695d51f6-d6fa-4bbd-bac0-ee8d3b895c8b.png">
<img width="653" alt="Screenshot 2022-11-04 at 15 46 10" src="https://user-images.githubusercontent.com/8542534/199987757-86e9ee47-2b3e-46e5-9f08-7f7127b006ce.png">
